cmd/truss/main.go

```diff
@@ -175,10 +181,11 @@ func parseInput() (*truss.Config, error) {
 
 	svcName = strings.ToLower(svcName)
 
-	svcDirName := svcName + "-service"
+	//svcDirName := svcName + "-service"
+	svcDirName := ""
```

Comment:
if you're trying to get the service to generate in the current directory, no codechange is needed. you can pass `--svc-out=./`